### PR TITLE
CVE-2022-24775: Guzzle PSR-7

### PR DESCRIPTION
**Pull request by Nyholm**
I just requested a CVE from Github. I guess they will come back next week.

### CODE REVIEW

**Review by GrahamCampbell — APPROVED** [review]

File: guzzlehttp/psr7/GHSA-q7rv-6hp3-vh96.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+title:     Inproper parsing of HTTP headers
+link:      https://github.com/guzzle/psr7/releases/tag/2.1.1
```
Comment (naderman) [inline guzzlehttp/psr7/GHSA-q7rv-6hp3-vh96.yaml:2]: Can you link to the advisory instead as it has more info?

```suggestion
link:      https://github.com/guzzle/psr7/security/advisories/GHSA-q7rv-6hp3-vh96
```

**Review by naderman — CHANGES_REQUESTED** [review]

File: guzzlehttp/psr7/GHSA-q7rv-6hp3-vh96.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+cve:       GHSA-q7rv-6hp3-vh96
```
Comment (naderman) [inline guzzlehttp/psr7/GHSA-q7rv-6hp3-vh96.yaml:3]: This is not a CVE id. Please do not list one if there isn't one. I think @nicolas-grekas previously used one with XXX placeholders which i can live with, but this is going to cause issues with scripts processing this automatically. Should probably add something to the CI script for this? cc @stof

**Comment by naderman** [comment]
Please use an appropriate filename as per https://github.com/FriendsOfPHP/security-advisories#contributing if there is no cve I'd yet, use the date format and you can rename the file once there is a cve id.

File: guzzlehttp/psr7/GHSA-q7rv-6hp3-vh96.yaml (hunk 2 above)
Comment (stof) [reply guzzlehttp/psr7/GHSA-q7rv-6hp3-vh96.yaml:3]: @naderman see #623

**Comment by Nyholm** [comment]
Thank you for the quick review. I got a CVE now and the PR is updated

File: guzzlehttp/psr7/GHSA-q7rv-6hp3-vh96.yaml (hunk 2 above)
Comment (Nyholm) [reply guzzlehttp/psr7/GHSA-q7rv-6hp3-vh96.yaml]: Thank you. I will remember this for next time

**Review by stof — APPROVED** [review]

**Comment by Nyholm** [comment]
Thank you for merging b

**Pull request merged**